server/tests/batch_comparison/test_sample_extra_batches_by_counting_group.py

```diff
@@ -113,8 +114,17 @@ def batch_tallies(
     assert_ok(rv)
 
 
+@pytest.mark.parametrize(
+    "org_id",
+    [
+        "TEST-ORG/sample-extra-batches-by-counting-group/automatically-end-audit-after-one-round",
+        "TEST-ORG/sample-extra-batches-by-counting-group",
+    ],
+    indirect=True,
+)
```

Comment:
What I used as a reference to figure out how to pass a parameter to a fixture function: https://stackoverflow.com/questions/18011902/how-to-pass-a-parameter-to-a-fixture-function-in-pytest